recipe/meta.yaml

```diff
@@ -1,12 +1,15 @@
 {% set name = "pymeep" %}
-{% set version = "1.16.1" %}
-{% set sha256 = "b0dc3c0090a6562eb67ad1dd986a3347dc188c0816dce3c5ec1e8870428d41d3" %}
-{% set buildnumber = 0 %}
+{% set version = "1.17.1" %}
+{% set sha256 = "cea11aafe8be1d7ab762324d2dc7d9f4a9b713cee87f9722ced2c5cc30f53a99" %}
+{% set buildnumber = 1 %}
 # recipe-lint fails if mpi is undefined
-{% set mpi = mpi or 'nompi' %}
+{% set mpi = mpi or "nompi" %}
 {% if mpi == "nompi" %}
 # prioritize nompi via build number
 {% set buildnumber = buildnumber + 100 %}
+{% set mpi_prefix = "nompi" %}
+{% else %}
+{% set mpi_prefix = "mpi_" + mpi %}
 {% endif %}
 
 package:
```

Comment:
Try using a different global name so it does not collide with the subpackages names. It has led to surprising behavior in the past, and might help here. Some naming conventions people usually follow is `name: {{ name }}-split`.